### ISSUE TITLE
Textarea with zero height and width cannot be interacted with via keyboard

### ISSUE THREAD

**Issue by greyhillman** [SeleniumHQ/selenium#6256]
## Meta -
OS:  Windows 10

Selenium Version:  IE Driver 3.13.0.0

Browser:  Internet Explorer

Browser Version:  IE 11.492.16299.0 64 bit

## Expected Behavior -
Entering text into a `textarea` with zero width and height should work, as it does in the other browsers.

## Actual Behavior -
WebDriver throws:
```
Element cannot be interacted with via the keyboard because it is not displayed.
```

## Steps to reproduce -

```html
<html>
<body>
<textarea style="height: 0px; width: 0px"></textarea>
</body>
</html>
```

**Comment by greyhillman** [comment]
This might be related to #6038.

**Comment by lmtierney** [comment]
Steps to reproduce need more than just a small HTML snippet. Please provide some code. Are you using regular SendKeys, are you using Actions?

**Comment by greyhillman** [comment]
Sorry, about not having some code. I didn't know what language to use.

In the project I'm working on, I'm using [WebDriverIO](http://webdriver.io/). They're essentially a wrapper over some WebDriver commands.

The testing code I have is:
```ts
describe("test", () => {
    it("test", () => {
        browser.url("https://microsoft.github.io/monaco-editor/playground.html");
        browser.pause(3000);

        browser.setValue('//textarea[1]', "test");
        browser.pause(3000);
    });
});
```
`browser.url` loads the pause, `browser.pause` pauses, and `browser.setValue` sends the second argument to input into the element with the given selector.

Here's the WebDriver commands sent to do `setValue`:
![image](https://user-images.githubusercontent.com/4807442/43851439-92172a30-9aef-11e8-8356-d737298ad0ae.png)

The main issue is with the monaco-editor. I've embedded it into a page and have not been able to set the value of the editor. I thought the issue was the `textarea` having 0 area, but I've been having issues trying to replicate the exact error I get on my page. The error obtained for the above code is not the same as my original one, but I think it's close enough. I can update the title if you want.
I've filed an issue (Microsoft/monaco-editor#994) in the monaco-editor seeing if having zero area is the actual issue and not the IE driver.
